packages/shared/src/components/comments/MainComment.tsx

```diff
@@ -43,6 +45,7 @@ export default function MainComment({
         <EnableNotification
           className={!comment.children?.edges?.length && 'mt-3'}
           source={NotificationPromptSource.NewComment}
+          contentName={user?.id !== comment?.author.id ? comment?.author?.name : undefined}
```

Comment:
@sshanzel I think this was removed by mistake before

Reply:
I don't really remember anything related to this. Could have gotten lost along the way by mistake through merge. Apologies for that as I was the one who did it. Will carefully merge bigger things on the next ones 🙏